### PR TITLE
Release google-cloud 0.59.0

### PR DESCRIPTION
**Pull request by quartzmo**
* Add google-cloud-security_center.
* Add google-cloud-phishing_protection.
* Add google-cloud-recaptcha_enterprise.

This pull request was generated using releasetool.

### CODE REVIEW

File: google-cloud/lib/google/cloud/version.rb [hunk 1]
```diff
@@ -15,6 +15,6 @@
 
 module Google
   module Cloud
-    VERSION = "0.58.0".freeze
+    VERSION = "0.2.0".freeze
```
Comment (blowmage) [inline google-cloud/lib/google/cloud/version.rb]: Needs to be `0.59.0`.

Comment (quartzmo) [reply google-cloud/lib/google/cloud/version.rb]: Fixed manually. This appears to be a bug in releasetool.

**Comment by busunkim96** [comment]
@quartzmo @blowmage Just wanted to check that the correct version here is v0.59.0. 

(I think releasetool got upset about the branch version number being 0.2.0)

**Comment by quartzmo** [comment]
@busunkim96 Yes, the correct version is `0.59.0`. I ran releasetool locally to create the commit and branch. I entered `minor` in the interactive CLI session and was surprised when the version (and branch name) were `0.2.0` instead of `0.59.0`. (Obviously 58 is followed by 59, right?) So I added a commit to the branch, "Fix new version manually", above. Do you have any idea why releasetool incremented the minor version to `2` instead of `59`? And what can we do now to unblock the release?

**Comment by busunkim96** [comment]
One way to unblock this would be to create a branch called `release-google-cloud-v0.59.0` and open a PR. You can then add add the label `autorelease: pending`. Autorelease should treat it as a release PR once it's been merged. 

@TheRoyalTnetennba may have better ideas, since he did the Kokoro setup on this repo.

**Comment by quartzmo** [comment]
@busunkim96 Thank you. We'll try that tomorrow.

@TheRoyalTnetennba Should this release wait until google-cloud-webrisk is added to google-cloud? google-cloud-webrisk was released in #3098 a few weeks ago, but it was never added as a google-cloud (or gcloud) dependency.

**Comment by busunkim96** [comment]
:wave: Would someone mind removing the `autorelease: pending` label from this PR? Our robot keeps trying to release it and failing. :robot:

**Pull request merged**